indexers/keyvalue/MongoDBIndexer/mongodbhandler.py

```diff
@@ -83,3 +87,30 @@ def __exit__(self, exc_type, exc_val, exc_tb):
             self.client.close()
         except pymongo.errors.PyMongoError as exp:
             raise MongoDBException(exp)
+
+    def delete(self, keys: Iterator[int], *args, **kwargs):
+        import pymongo
+        try:
+            count = self.collection.delete_many({'_id': {'$in': keys}}).deleted_count
+            self.logger.debug(f'deleted {count} documents in the database')
+        except pymongo.errors.PyMongoError as exp:
+            self.logger.error(f'got an error while deleting a document in the db {exp}')
+
+    def update(self, keys: Iterator[int], values: Iterator[bytes], ):
```

Comment:
add *args and **kwargs as per discussion here jina-ai/jina#1285

Reply:
Done